src/background/Wallet/Wallet.ts

```diff
@@ -1065,7 +1095,11 @@ export class Wallet {
     const txWithFee = await prepareGasAndNetworkFee(prepared, networks, {
       source: mode === 'testnet' ? 'testnet' : 'mainnet',
     });
-    const transaction = await prepareNonce(txWithFee, networks, chain);
+    // todo: remove after update to ethers v6
```

Comment:
Let's write more clearly which line exactly should be removed, otherwise it'll be hard to remember